crates/moonbuild-rupes-recta/src/build_plan/mod.rs

```diff
@@ -68,7 +75,14 @@ impl BuildPlan {
         self.spec.get(&node)
     }
 
-    pub fn incoming_nodes(&self, node: BuildPlanNode) -> impl Iterator<Item = BuildPlanNode> + '_ {
+    /// Get the list of nodes that **the given node depends on**.
+    pub fn depend_on_nodes(&self, node: BuildPlanNode) -> impl Iterator<Item = BuildPlanNode> + '_ {
```

Comment:
NB: The [crates.io](https://crates.io/crates/serde) convention gives dependency/dependent.